dbt_meshify/dbt.py

```diff
@@ -27,7 +28,12 @@ def invoke(
 
     def parse(self, directory: os.PathLike):
         logger.info("Executing dbt parse...")
-        return self.invoke(directory, ["--quiet", "parse"])
+        try:
+            return self.invoke(directory, ["--quiet", "parse"])
+        except UninstalledPackagesFoundError:
+            logger.debug("Project missing packages, installing...")
+            self.invoke(directory, ["deps"])
+            return self.invoke(directory, ["--quiet", "parse"])
```

Comment:
This is _very_ clever! 🤔 Does this exception ever get raised for other `Dbt` methods? If so, it may make sense to wrap/decorate `invoke()` directly to handle this exception for all dbt calls. Thoughts, @dave-connors-3?

Reply:
ah great point! I think it's around for any submitted command when packages aren't installed, but I put it in `parse` since it's always the first method we call. I can add it up a level to `invoke` to be safer!